apps/vqa-service/src/query.py

```
@@ -99,7 +99,10 @@ def tryConvertToJSON(response):
             # Try finding everything in between brackets
             match = re.search(r"(\{[\w\W]*\})", cleaned)
             if match:
-                return json.loads(match.group(1))
+                try:
+                    return json.loads(match.group(1))
+                except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)